packages/junon-io/server/entities/event_handler.js

```diff
@@ -622,6 +622,15 @@ class EventHandler {
 
     return entity.getTypeName()
   }
+  
+  getEffect(entityId) {
+    let player = this.getPlayer(entityId)
+    if (player) {
+      return player.hasEffect()
```

Comment:
wrong function. this woulnt work

Reply:
sorry about that I think this updated one should work